platform/blueprint/builtin/db.go

```diff
@@ -81,6 +81,14 @@ func (db *DB) Save(bp *blueprint.Blueprint) error {
 	if err != nil {
 		return err
 	}
+	check_bp, err := db.BlueprintByName(bp.Name)
+	if err != nil {
+		if !isNotFound(err) {
+			return err
+		}
+	} else if bp.UUID != check_bp.UUID {
```

Comment:
You can drop the else if you make the change above since you `return`

Given all the possible branches here, do you think a switch statement would make this logic easier to reason about?

Reply:
Drop the else, and just have two if statements? Sure, I suppose.

Reply:
Now we're doing an unnecessary comparison by eliminating the else.

Reply:
I switched it back because I think its correct that way. Happy to discuss. If you're good with it let me know and I can merge. :)